src/db_ledger.rs

```diff
@@ -830,11 +830,16 @@ impl Iterator for EntryIterator {
     }
 }
 
-pub fn create_empty_ledger(ledger_path: &str, genesis_block: &GenesisBlock) -> Result<()> {
+pub fn create_new_ledger(ledger_path: &str, genesis_block: &GenesisBlock) -> Result<(u64, Hash)> {
     DbLedger::destroy(ledger_path)?;
-    DbLedger::open(ledger_path)?;
     genesis_block.write(&ledger_path)?;
-    Ok(())
+
+    // Add a single tick linked back to the genesis_block to bootstrap the ledger
+    let db_ledger = DbLedger::open(ledger_path)?;
+    let entries = crate::entry::create_ticks(1, genesis_block.last_id());
+    db_ledger.write_entries(DEFAULT_SLOT_HEIGHT, 0, &entries)?;
```

Comment:
This is great because formerly the bank didn't count the beginning entries in the genesis block as ticks, which required the db_ledger also not count those as ticks. Glad we can get rid of that special case logic :P